### PR TITLE
Avalanche

### PR DESCRIPTION
**Pull request by iris112**

### CODE REVIEW

File: packages/limit-orders-react/src/utils/getTokenList.ts [hunk 1]
```diff
@@ -58,7 +60,9 @@ export default async function getTokenList(
     }
 
     const json = await response.json();
+    console.log(json);
```
Comment (gitpusha) [inline packages/limit-orders-react/src/utils/getTokenList.ts:63]: Remove console.log

File: packages/limit-orders-react/src/utils/getTokenList.ts [hunk 2]
```diff
@@ -58,7 +60,9 @@ export default async function getTokenList(
...
     if (!tokenListValidator(json)) {
+      console.log(tokenListValidator.errors);
```
Comment (gitpusha) [inline packages/limit-orders-react/src/utils/getTokenList.ts:65]: remove

File: packages/limit-orders-react/src/utils/getTokenList.ts (hunk 1 above)
Comment (iris112) [reply packages/limit-orders-react/src/utils/getTokenList.ts]: Fixed

File: packages/limit-orders-react/src/utils/getTokenList.ts (hunk 2 above)
Comment (iris112) [reply packages/limit-orders-react/src/utils/getTokenList.ts]: Fixed

**Pull request merged**